# Get current EGM state

---
bool success
int32 mode
string msg
